Source/carthage/Extensions.swift

```diff
@@ -55,21 +56,50 @@ extension GitURL: ArgumentType {
 	}
 }
 
+extension Color.Wrap {
+	func autowrap(string: String) -> String {
+		return Formatting.color ? self.wrap(string) : string
+	}
+}
+
+internal struct Formatting {
+	static let color = Terminal.isTTY && !Terminal.isDumb
+	
+	static let bulletin = Color.Wrap(foreground: .Blue, style: .Bold)
+	static let bullets: String = {
+		return bulletin.autowrap("***") + " "
+	}()
+	
+	static let URL = [StyleParameter.Underlined] as Color.Wrap
+	static let projectName = [StyleParameter.Bold] as Color.Wrap
+	static let path = Color.Wrap(foreground: .Yellow)
+	
+	static func quote(string: String) -> String {
+		return Color.Wrap(foreground: .Green).autowrap("\u{0022}" + string + "\u{0022}")
```

Comment:
@jspahrsummers Mostly it's just the syntax highlighting in Xcode.

![screen shot 2015-01-29 at 10 10 32 pm](https://cloud.githubusercontent.com/assets/782837/5970710/f6b06bc8-a804-11e4-8e92-238c51058994.png)

Backslash escaped special characters don't have a differentiating highlight color. Seeing two-plus same-colored double-quotes in one valid string literal always makes me pause reflexively…

It definitely trades intuitiveness for scanability though, I'd be happy to change it back :dolphin: